stardog/metadata.csv

```diff
@@ -0,0 +1,52 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+stardog.dbms.mem.mapped.max,gauge,,byte,,The maximum amount of memory used for memory mapping,0,stardog,Max Mem Map
+stardog.dbms.mem.mapped.used,gauge,,byte,,The number of bytes used in the memory map,-1,stardog,Mapped Mem Used
+stardog.dbms.memory.direct.reserve,gauge,,byte,,The amount of direct memory reserved,0,stardog,Direct Mem Reserved
+stardog.system.cpu.usage,gauge,,byte,,CPU usage,-1,stardog,CPU Usage
+stardog.system.uptime,gauge,,seconds,,The number of seconds that Stardog has been running,0,stardog,Uptime
+stardog.dbms.memory.heap.reserve,gauge,,byte,,The heap memory reserved,0,stardog,Heap Reserve
+stardog.dbms.memory.managed.direct,gauge,,byte,,Managed direct memory,0,stardog,Managed Direct Mem
+stardog.dbms.memory.managed.heap,gauge,,byte,,Managed heap memory,0,stardog,Managed Heap Mem
+stardog.dbms.mem.heap.used,gauge,,byte,,The amount of heap memory used,0,stardog,Heap Mem Used
+stardog.dbms.mem.heap.max,gauge,,byte,,The maximum heap size,0,stardog,Max Heap Size
+stardog.dbms.mem.direct.buffer.used,gauge,,byte,,The amount of direct memory used,-1,stardog,Direct Mem Used
+stardog.dbms.mem.direct.buffer.max,gauge,,byte,,The maximum amount of direct memory that can be used,0,stardog,Direct Mem Max
+stardog.dbms.mem.direct.pool.max,gauge,,byte,,The maximum size of the direct memory pool,-1,stardog,Direct Pool Max
+stardog.dbms.mem.direct.pool.used,gauge,,byte,,The amount of memory used from the direct pool,-1,stardog,Direct Pool Used
+stardog.dbms.mem.direct.max,gauge,,byte,,The maximum amount of direct memory that can be used,0,stardog,Direct Mem Max
+stardog.dbms.mem.direct.used,gauge,,byte,,The amount of direct memory used,-1,stardog,Direct Mem Used
+stardog.databases.system.planCache.ratio,gauge,,percent,,Plan cache ration,0,stardog,Plan Cache Ratio
+stardog.databases.system.planCache.size,gauge,,bytes,,Plan cache size,0,stardog,Plan Cache Size
+databases.<database name>.txns.openTransactions,gauge,,unit,,The number of open transations on a given database,0,stardog,Open TX
```

Comment:
Two notes here:
1) This metric (and all those below) should also be prefixed by `stardog.`, otherwise it could be very hard to find these metrics
2)  We don't use variable metric names because they make dashboards really hard to build - the database name should be added as a tag instead, that way users can easily aggregate and compare data from across different databases. For example, the ideal use case here would be for someone to be able to simply look up `stardog.databases.txns.openTransactions` and then use tags to sort out data from the database(s) they want to look at

Reply:
Thanks @ISauve.  For some of the metrics listed we don't have a way to get them across the entire system and thus they are only available when scoped to a database.  Is there an elegant way to handle that situation?

Reply:
If I'm correct in understanding that you're referring to the metrics which are database-specific (`txns.openTransactions`, `.txns.speed`, `.queries.running`, `.queries.speed`, `.openConnections`) then what you've done here is perfect. This ideal solution is for the metric itself to be generalized so that it will appear if the data is present for _any_ database, and then the tag with the database name gives the specific scope of the data - which is exactly what you did!